lib/ProductOpener/APIProductWrite.pm

```diff
@@ -85,6 +86,22 @@ sub skip_protected_field ($product_ref, $field, $moderator = 0) {
 	return 0;
 }
 
+=head2 is_protected_image($product_ref, $imagefield)
```

Comment:
Could you move this function to Images.pm instead? It's not specific to the API (it's useful to know if an image is protected on the website as well, for instance we could display it to users when they edit the product page)

Reply:
Where can we display it to users at the product edit page, maybe we could add a message below the image when user tries to crop or unselect an image?

Reply:
Very good idea. On the product edit page, we should completely hide the part where you can select or upload a new image, and instead display a message like "An image has been sent by the manufacturer. If you think it is incorrect or not up-to-date, please contact us."
![image](https://github.com/openfoodfacts/openfoodfacts-server/assets/8158668/60310f0e-5288-470c-81ff-5d85db51c0d5)